mailbox/lucene/src/test/java/org/apache/james/mailbox/lucene/search/LuceneMailboxMessageSearchIndexTest.java

```diff
@@ -56,14 +56,18 @@
 import org.junit.Test;
```

Comment:
Do it in AbstractMessageSearchIndexTest for all backends please. Thus even less code is needed